src/openstudio_lib/RunTabView.cpp

```diff
@@ -171,9 +171,9 @@ RunView::RunView() : QWidget(), m_runSocket(nullptr) {
 
 void RunView::onOpenSimDirClicked() {
   std::shared_ptr<OSDocument> osdocument = OSAppBase::instance()->currentDocument();
-  QString url = QString::fromStdString((getCompanionFolder(toPath(osdocument->savePath())) / toPath("run")).string());
-  QUrl qurl = QUrl::fromLocalFile(url);
-  if (!QDesktopServices::openUrl(qurl)) {
+  path runDir = getCompanionFolder(toPath(osdocument->savePath())) / toPath("run");
+  QString path = QDir::toNativeSeparators(toQString(runDir));
```

Comment:
FYI @j-lorrey : the issue here is that the OpenStudio path class on Windows uses UTF-16 encoding, we were passing that to `QString::fromStdString` which expects a UTF-8 string.  The fix is to use [toQString](https://github.com/openstudiocoalition/OpenStudioApplication/blob/develop/src/model_editor/Utilities.cpp#L77) to convert the path, that method handles conversion from UTF-16 on Windows.